tools/find-regression.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../.github/scripts/config.sh
+
+REPOSITORY=$1
+REGRESSION_COMMIT=origin/rebase-upstream
+STABLE_COMMMIT=origin/woarm64
+FEATURE_BRANCH=origin/woarm64
+
+pushd $SOURCE_PATH/$REPOSITORY
+    git bisect start
+    git bisect bad $REGRESSION_COMMIT
```

Comment:
My repos in `code` folder don't know about other remote branches than `woarm64`. So I got this error:

```
+ git bisect bad origin/rebase-upstream
error: Bad rev input: origin/rebase-upstream
```

From what I tried it's because we use `--single-branch` in `update-source.sh`. After running `git remote set-branches --add origin rebase-upstream` and `git fetch`, I have the `origin/rebase-upstream` branch there

Reply:
Yes. I don't think it's responsibility of this script to handle this case as it will be run from daily updates workflow where full repository is forked. You've found the solution already. Alternative one is to change `GCC_BRANCH` in `config.sh` to `rebase-upstream` and then run `.github/scripts/update-repositories.sh`.

Reply:
Ok, since the plan is to run it in workflow, it's fine for me to keep it as is.

Reply:
Maybe we can do single branch clone during the workflow build while normal clone when building locally, WDYT?

Reply:
What would be the benefits?

Afaik single branch clone saves some time but it's disadvantage is that you need to handle missing branches if scripts like this one needs another branch. It seems to me that both advantage and disadvantage hold in workflows as well as locally. So why not having the same approach in workflow and locally?

Reply:
My point of view is that in the local build scenario, the clonning is done once and then switching the branches several times. In workflow scenario, the clone is done every single time and branch is never switched.

Reply:
I see. I would still prefer to use one approach in both scenarios but what you suggest also makes sense to me.

Reply:
OK, let's keep it as-is.